modules/distr-tracing-rn-new-features.adoc

```diff
@@ -11,11 +11,13 @@ Result – If changed, describe the current user experience.
 [id="distr-tracing-rn-new-features_{context}"]
 == New features and enhancements {ProductName} 2.0.0
 
-This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}.  This effort includes the following:
+This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}. This name change is to reflect that the distributed tracing solution encompasses more than just Jaeger project components, but also Elasticsearch, AMQ Streams, and OpenTelemetry. The rebranding also allows for new technologies and components to be added in the future under the distributed tracing name.
```

Comment:
I don't know if I would mention Elasticsearch and AMQ stream here. Those are components which we can use with our distributed tracing solution, but they are their own separate components. Elasticsearch and AMQ streams are also not something new with this release or as part of our rebranding.